lib/widgets/element_on_list.dart

```diff
@@ -0,0 +1,61 @@
+import 'package:flutter/material.dart';
+
+import '../classes/news.dart';
+import '../Screens/specific_story.dart';
+
+class ElementOnList extends StatefulWidget {
+  Future<News> specificNews;
+  ElementOnList({super.key, required this.specificNews});
+
+  @override
+  State<ElementOnList> createState() => _ElementOnListState();
+}
+
+class _ElementOnListState extends State<ElementOnList> {
+  @override
+  Widget build(BuildContext context) {
+    return GestureDetector(
+      onTap: () {
+        Navigator.push(
+            context,
+            MaterialPageRoute(
+              builder: (context) => SpecificNews(news: widget.specificNews),
+            ));
+      },
+      child: Padding(
+        padding: const EdgeInsets.all(8.0),
+        child: FutureBuilder<News>(
+            future: widget.specificNews,
+            builder: ((BuildContext context, AsyncSnapshot<News> snapshot2) {
```

Comment:
Here and everything below looks like it can move over a smidge.
```suggestion
          future: widget.specificNews,
          builder: ((BuildContext context, AsyncSnapshot<News> snapshot2) {
```

Reply:
Line 57 below is likely missing some trailing commas to get the formatter to do the correct formatting here.